src/components/attendee/ProfileOverlay/ProfileOverlay.module.scss

```
@@ -9,7 +9,7 @@
     max-width: 800px;
     min-height: 100vh;
     margin: 0 auto 0 0;
-    padding: 0 $whitespace * 3 $whitespace * 4;
+    padding: 0 space.empty(1) * 3 space.empty(1) * 4;
```

This line and similar to it are the whole reason for existence of `space.empty()` as a function. 
It makes clear which part is being manipulated in case you encounter multiple factors (`a * b * c * d`)

```suggestion
    padding: 0 space.empty(3) space.empty(4);
```

---

[7f5a37b](https://github.com/sparkletown/sparkle/pull/2840/commits/7f5a37baa83617db540f69b073ae690c37f12ce6)